libs/@guardian/source-react-components/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @guardian/source-react-components
 
+## 19.1.0
+
+### Minor Changes
+
+- 6603b42: Reduces text size of `xsmall` buttons to `14px`.
```

Comment:
is this a minor change?

>Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backward compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code.

https://semver.org/#spec-item-7

Reply:
We've decided this should be `major` after all, but are going to look into how other design systems handle component versioning and come up with some guidance so that we're consistent and there are no surprises for anybody using these packages.